app/views/index.erb

```diff
@@ -1 +1,21 @@
-<h1>Welcome!</h1>
+<div class="container-fluid">
+  <!-- <button type="button" class="btn btn-primary btn-lg">Updates</button> -->
+  <img src="/vendor/bootstrap/images/bikes.jpg" class="img-responsive" alt="Responsive image">
+</div>
+
+<div class='container'>
+  <div class='row'>
+    <div class='col-md-4'>
+      <h2>Stations</h2>
+      <p>PLACE DE LA FONTAINE STRAVINSKY QUAI DE VALMY RUE D’ULM - PLACE DU PANTHEON 750 BOOST GREY WITH GUM SOLE HYATT REGENCY HOTEL DRIVE AND REUNION BLVD EAST @ 10:30PM KEATING DRIVE AND COTTON BOWL CIRCLE @ 11:30PM I forgot to add Daniel Ek</p>
```

Comment:
We should probably figure out what type of descriptions we want to put here. But I'll actually play around with that tonight.